app/views/customers/edit.html.erb

```diff
@@ -1,10 +1,3 @@
 <h1>Editing customer</h1>
 
-<%= render "form", customer: @customer %>
-
-<br>
-
-<div>
-  <%= link_to "Show this customer", @customer %> |
-  <%= link_to "Back to customers", customers_path %>
-</div>
```

Comment:
In terms of UX, it's a bit strange that I need to use the navigation bar or browser's Back button to go back to the list view now, but I wouldn't consider this to be important for this assignment. 😅

Reply:
My logic of user movement was that he always returns to the main page (also the staff page) after every action of creation or deleting. On the root page I placed every controls to use current application, so I thought that these buttons were not needed ;-)
So here you can say: "I'm an artist, that's how I see it" (joke)

Reply:
It's not uncommon to go into an Edit view, and then change your mind about it. In those cases, it's better for the web application to have a safe way of returning to the list without risking any changes.